.circleci/config.yml

```diff
@@ -6,19 +6,25 @@ jobs:
       - image: oasislabs/rust:latest
     environment:
         PATH: /root/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+        DEBIAN_FRONTEND: noninteractive
     steps:
       - checkout
       - run:
           name: Install tools
-          command: cargo install oasis-build oasis-cli --debug
+          command: |
+            cargo install oasis-build oasis-cli oasis-chain --debug
+            apt-get -qq update
+            apt-get -qq install nodejs npm
       - run:
           name: Build service
           working_directory: hello-world/service
           command: oasis build
       - run:
           name: Test app
           working_directory: hello-world/app
-          command: oasis test
+          command: |
+            oasis-chain &
+            oasis test
```

Comment:
you can remove this once you replace the service with a submodule that's already tested by another CI